SlowCheetah/Logging/XmlTransformationPreviewLogger.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Sayed Ibrahim Hashimi. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See  License.md file in the project root for full license information.
+
+namespace SlowCheetah
+{
+    using System;
+    using Microsoft.Web.XmlTransform;
+
+    /// <summary>
+    /// Shim for using an inernal <see cref="ITransformationLogger"/> in <see cref="XmlTransformer"/>
+    /// </summary>
+    public class XmlTransformationPreviewLogger : IXmlTransformationLogger
```

Comment:
As we discussed offline, create a single shim implementation that goes from `ITransformationLogger` to `IXmlTransformationLogger`. Don't have a shim per logger.